src/test/test_lib.ts

```diff
@@ -28,11 +23,11 @@ test('createClassValidator creates an ajv ValidateFunction for each method signa
     },
   };
   const validators = createClassValidator(schema, 'Foo', 'params');
-  expect(validators.hello({ name: 'heh' })).to.be.true;
-  expect(validators.hello({})).to.be.false;
+  t.true(validators.hello({ name: 'heh' }));
```

Comment:
`think(Reflect.selfReference()).that.tHis.is(more(readable))`

Reply:
lolz